ui/ui-community/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Hmm, this changed with the latest rebase. I wonder why 🤔

Reply:
🤷 Next changes these things automatically from time to time...